### PR TITLE
PE-1886: Integrate with Advanced Metadata (IPFS)

### PR DESCRIPTION
**Pull request by matibat**
Changes:
- Integrates the alpha version of CORE 
- Appends the IPFS CID to the custom metadata of each uploaded file

### CODE REVIEW

File: src/commands/upload_file.ts [hunk 1]
```diff
@@ -121,6 +142,32 @@ function getSingleFile(parameters: ParametersHelper, parentFolderId: FolderID):
 	return [singleParameter];
 }
 
+async function getCustomMetaDataWithIpfsCid({
```
Comment (fedellen) [inline src/commands/upload_file.ts:145]: Is it possible to handle this in the parameter helper?

**Review by fedellen — APPROVED** [review]
LGTM, left non-blocking question. Only because we can have some unit tests more easily from there

File: src/commands/upload_file.ts (hunk 1 above)
Comment (fedellen) [reply src/commands/upload_file.ts:145]: Following up on this, perhaps `getCustomMetaData` would check for the `--ipfs-add` flag internally

Comment (matibat) [reply src/commands/upload_file.ts]: Moves the function to parameters helper: 3167dbc2ae8c54570ce919a11fd028dc53dce5f9
And adds unit tests: c703ac2ffb3f32cfe1be4c2a48a63a15b2fd6b8c

**Review by fedellen — APPROVED** [review]
Great updates!

**Pull request merged**